### ISSUE TITLE
bug / missing functionality: Request and Response Body Examples

### ISSUE THREAD

**Issue by ahmadnassri** [mrin9/RapiDoc#292]
https://swagger.io/docs/specification/adding-examples/#request-response-bodies

when using inline `example` properties it works fine, but when using `$ref` and exposing an example property on the top-level it shows an empty string:

```
content:
  application/json:
    schema:
      $ref: ../../shared/schemas/Error.yml
    example:
      type: https://example.com/probs/out-of-credit
      title: You do not have enough credit.
      detail: Your current balance is 30, but that costs 50.
      instance: "/account/12345/msgs/abc"
      balance: 30
      accounts:
      - "/account/12345"
      - "/account/67890"
```

![image](https://user-images.githubusercontent.com/183195/91775302-b87e0d80-ebb8-11ea-8c99-7d23a953f486.png)

**Comment by bkarvas** [comment]
This is the same issue as #259.

For better understanding of both issues, just update examples.yaml spec which is originally located at this link https://github.com/mrin9/RapiDoc/blob/master/docs/specs/examples.yaml
Added new path "/single-example-by-response-type2:" which is identical to your "/single-example-by-response-type:" except the sample part.

```
  /single-example-by-response-type2:
    get:
      summary: Single Example at schema Level
      tags:
        - Providing Examples
      responses:
        '200':
          description: Successful operation
          content:
            application/json:
              schema:
                type: object
                properties:
                  age:
                    description: Person Age
                    type: integer
                  fullName:
                    description: Person Full name
                    type: object
                    properties:
                      firstName:
                        description: First name
                        type: string
                      lastName:
                        description: Last name
                        type: string
              example:
                age: 10
                fullName:
                  firstName: Donald
                  lastName: Duck
```

**Comment by mrin9** [comment]
thanks... will take a look at it soon. our team is little held up for abt 2-3 weeks with an internal release

**Comment by ahmadnassri** [comment]
no worries, not a show blocker for me currently

**Comment by mrin9** [comment]
Please test it with the latest build from `dist` folder, and let us know if it takes care of your problem

**Comment by bkarvas** [comment]
Finaly solved even the issue #259.
This helped me a lot 👍

**Comment by rvdwijngaard** [comment]
This fix solved my problem too. Do you have plans for making an official release?
